packages/angular_devkit/build_angular/src/differential-loading/is-differential-loading-needed.ts

```diff
@@ -0,0 +1,138 @@
+/**
```

Comment:
Maybe this file is better names as `utils.ts`, what do you think?

Reply:
Coincidental cohesion is not ideal. I like the current name. Maybe `_` instead of `-` so we can be consistent?

Reply:
That is true, though I am not sure about the file name per see, since the exported methods here, are really not related to 'differential loading enabled' per see.

Such as `getScriptTarget`, also, I won’t name a file with an `is` prefix in that case. Unless this file has a single responsibility. Though still naming a module with an `is` prefix doesn’t seems right to me. imho.

Reply:
For me, both is ok. The only function we need to export is `isDifferentialLoadingNeeded`. The others are only exported for testability.

Reply:
it's now called util